### ISSUE TITLE
Display type information on variable hover

### ISSUE THREAD

**Issue by BladeMF** [phpactor/vscode-phpactor#11]
When hovering variables, type information should be displayed:

![image](https://user-images.githubusercontent.com/12109622/102048485-e2fe7a00-3de7-11eb-8cf7-31d6c57c385c.png)

**Comment by dantleech** [comment]
hmm, you should see this

![image](https://user-images.githubusercontent.com/530801/102056308-150ddd00-3de4-11eb-9074-9bae6e4aaa9a.png)

**Comment by BladeMF** [comment]
Nope. Nothing like this. I saw the twig that creates this.

**Comment by BladeMF** [comment]
But my code gets highlighted. Just no text. Not sure how to debug the calls to the language server.

**Comment by dantleech** [comment]
Checking the logs should help.

**Comment by BladeMF** [comment]
Not really:

```
[2020-12-14 10:18:51] phpactor.DEBUG: pid:18519 Find process "'find' '/root/temp' '-mindepth' '1' '-newercc' '/tmp/amp-fs-watchmtnZO1'" done in 0.00 seconds [] []
[2020-12-14 10:18:52] phpactor.DEBUG: Resolving: Microsoft\PhpParser\Node\Statement\ClassDeclaration [] []
[2020-12-14 10:18:52] phpactor.DEBUG: Resolving: Microsoft\PhpParser\Node\Parameter [] []
[2020-12-14 10:18:52] phpactor.DEBUG:  OK [0.0000] "Folder1\Class2" with locator "Phpactor\Extension\LanguageServerWorseReflection\SourceLocator\WorkspaceSourceLocator" [] []
[2020-12-14 10:18:52] phpactor.DEBUG: Resolving: Microsoft\PhpParser\Node\Expression\Variable [] []
[2020-12-14 10:18:52] phpactor.DEBUG: Resolving: Microsoft\PhpParser\Node\Expression\Variable [] []
[2020-12-14 10:18:52] phpactor.DEBUG: pid:20126 Find process "'find' '/root/temp' '-mindepth' '1' '-newercc' '/tmp/amp-fs-watchsTfKMN'" done in 0.00 seconds [] []
[2020-12-14 10:18:52] phpactor.DEBUG: Resolving: Microsoft\PhpParser\Node\Statement\ClassDeclaration [] []
[2020-12-14 10:18:52] phpactor.DEBUG: Resolving: Microsoft\PhpParser\Node\Parameter [] []
[2020-12-14 10:18:52] phpactor.DEBUG: Resolving: Microsoft\PhpParser\Node\Expression\Variable [] []
[2020-12-14 10:18:52] phpactor.DEBUG: Resolving: Microsoft\PhpParser\Node\Expression\Variable [] []
```

**Comment by BladeMF** [comment]
Can I force the calls to be logged?
Is `Phpactor\Extension\LanguageServerHover\Handler\HoverHandler` the class that should handle it?

**Comment by BladeMF** [comment]
sooo, ::hover gets called, but the Promise doesn't get executed.

**Comment by BladeMF** [comment]
Fixed in #18
